core/controllers/base_test.py

```diff
@@ -107,9 +120,55 @@ def test_redirect_in_both_logged_in_and_logged_out_states(self):
         self.login('user@example.com')
         response = self.testapp.get('/')
         self.assertEqual(response.status_int, 302)
+
+        # Homepage is either dashboard or library based on user's learner
+        # status, this homepage redirect rule behavior is seperately tested
+        atHome = ('dashboard' in response.headers['location'] or
```

Comment:
@thedeveloperr Yes, I would say that if a user has edited another user's explorations then we should consider that user a creator and redirect him or her to the creator dashboard.

@seanlip Do we have any evidence that a learner would ever attempt to edit an exploration? If so, then perhaps I should reconsider my thoughts. Then again, I suspect this will get looked at again when we implement the learner dashboard, and doing this for now is probably better than not doing it.